spec/models/user_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
-RSpec.describe User, type: :model do
+describe User, type: :model do
+  let(:username) { 'Factorial' }
+  let(:user) { FactoryGirl.create(:user, username: username) }
```

Comment:
Git won't let me tab on the line, but on line 11, you're still creating a user active record object (instead of using your factory). No longer necessary to do that line